portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -22,16 +22,19 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment");
```

Comment:
Another option to consider instead of keeping commented out code, is to add a response right at the start (potentially including a message about the API being disabled in the response); then immediately return. Wrapping the code in an "if false{}" type block can also accomplish this.